spec/lang/syntax.md

```diff
@@ -152,13 +154,20 @@ pub enum IntRel {
     Ne,
 }
 
+pub enum BinOpBool {
+    /// Bitwise-and on booleans.
+    BitAnd,
+} 
+
 pub enum BinOp {
     /// An operation on integers, with the given output type.
     Int(BinOpInt, IntType),
     /// A relation between integers.
     IntRel(IntRel),
     /// Pointer arithmetic (with or without inbounds requirement).
     PtrOffset { inbounds: bool },
+    /// An operation on booleans
```

Comment:
```suggestion
    /// An operation on booleans.
```

Reply:
You ended up adding this comment at a different place than where I suggested it. :)